src/Support/Concerns/UsesMonitoredScheduledTasks.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace Spatie\ScheduleMonitor\Support\Concerns;
+
+use Spatie\ScheduleMonitor\Support\ScheduledTasks\MonitoredScheduledTasks;
+
+trait UsesMonitoredScheduledTasks
+{
+    public function getScheduleMonitoringConfigurationsRepository(): MonitoredScheduledTasks
```

Comment:
Rename this to `getMonitoredScheduledTasks`

Reply:
Thank you, missed this one.

✅  Renamed